drivers/pds/lib/utils.go

```diff
@@ -1093,6 +1104,59 @@ func CreateTempNS(length int32) (string, error) {
 	return namespace, nil
 }
 
+func isReachbale(url string) (bool, error) {
```

Comment:
Also Can we make this a public function . This could be useful to other scripts as well

Reply:
done